hathor/transaction/storage/transaction_storage.py

```diff
@@ -541,6 +558,41 @@ def get_all_sorted_txs(self, timestamp: int, count: int, offset: int) -> Transac
         """
         raise NotImplementedError
 
+    def add_value(self, key: str, value: str) -> None:
+        """ Save value on storage
+            Need to be a string to support all storages, including rocksdb, that needs bytes
+        """
+        # XXX Some storages (e.g. memory storage) don't need this attribute
```

Comment:
I think we should raise a `NotImplementedError` here, and have an empty method in the storages that don't need this attribute. By the way, why don't we need it in the memory storage?

Reply:
Added to memory and remote storage. It was my mistake because the previous method were not needed there. Can you see if it's good?

Reply:
It is. Thanks!